src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -0,0 +1,230 @@
+
+The following options have been replaced by more comprehensive configuration, covered in the section linked above:
+
+* `apiServerSecurePort`
+* `etcdTlsSecretName`
+* `etcdTlsSecretNamespace`
+* `controllerManagerEndpointUrl`, `etcdEndpointUrl`, `apiServerEndpointUrl`, and `schedulerEndpointUrl`
+
+### Agent configuration [#agent-configuration]
+
+Agent config file, previously specified in `config` has been moved to `common.agentConfig`. Format of the file has not changed, and the full range of options that can be configured can be found [here](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/).
+
+The following agent options were previously "aliased" in the root of the `values.yml` file, and are no longer available:
+
+* `logFile` has been replaced by `common.agentConfig.log_file`.
+* `eventQueueDepth` has been replaced by `common.agentConfig.event_queue_depth`.
+* `customAttributes` has changed in format to a yaml object. The previous format, a manually json-encoded string e.g. `{"team": "devops"}`, is still accepted although discouraged.
+* Previously, `customAttributes` had a default `clusterName` entry which might have unwanted consequences if removed. This is no longer the case, users may now safely override `customAttributes` on its entirety.
+* `discoveryCacheTTL` has been completely removed, as the discovery is not performed using kubernetes informers.
+### Integrations configuration [#integrations-configuration]
+
+Integrations were previously configured under `integrations_config`, using an array format:
+
+```yaml
+integrations_config:
+  - name: nri-redis.yaml
+    data:
+      discovery:  # ...
+      integrations:  # ...
+```
+
+The mechanism remains the same, but we have changed the format to be more user-friendly:
+
+```yaml
+integrations:
+  nri-redis-sampleapp:
+    discovery:  # ...
+    integrations:  # ...
+```
+
+Moreover, it is now mandatory to use the `--port` and `--tls` flags on the discovery command. Before, the following would work:
+
+```yaml
+integrations:
+  nri-redis-sampleapp:
+    discovery:
+      command:
+        exec: /var/db/newrelic-infra/nri-discovery-kubernetes
+```
+
+From v3 onwards, you must specify `--port` and `--tls`:
+
+```yaml
+integrations:
+  nri-redis-sampleapp:
+    discovery:
+      command:
+        exec: /var/db/newrelic-infra/nri-discovery-kubernetes --tls --port 10250
+```
+
+This is change is required because in v2 and below, the `nrk8s-kubelet` component (or its equivalent) ran with `hostNetwork: true`, so `nri-discovery-kubernetes` could connect to the kubelet using `localhost` and plain http. For security reasons, this is no longer the case, hence the need to specify both flags from now on.
```

Comment:
Typo "This is change is"

Reply:
https://github.com/newrelic/docs-website/pull/5742/commits/c884563db3ca965db7f5f0b3d559295b3d8972e2